config_db.go

```diff
@@ -133,18 +136,97 @@ func byIDs(ids []string) bson.M {
 	return bson.M{idKey: bson.M{"$in": ids}}
 }
 
-func getSectionsBSON(ctx context.Context, ids []string) ([]bson.Raw, error) {
-	cur, err := GetEnvironment().DB().Collection(ConfigCollection).Find(ctx, byIDs(ids))
+func getSectionsBSON(ctx context.Context, ids []string, includeOverrides bool) ([]bson.Raw, error) {
```

Comment:
i feel like this function took me a bit to figure out what was going on 
could we comment the function so it's easier to read!

Reply:
done